advisories/2.1.0/BRSA-w8ZsgmV2.toml

```diff
@@ -0,0 +1,20 @@
+[advisory]
+id = "BRSA-w8ZsgmV2"
+title = "kernel CVE-2024-36971"
+cve = "CVE-2024-36971"
+severity = "moderate"
+description = "In the Linux kernel, the following vulnerability has been resolved: -  - net: fix __dst_negative_advice() race"
+
+[[advisory.products]]
+package-name = "kernel-5.10"
+patched-version = "kernel-5.10.219-208.866.amzn2"
+
+[[advisory.products]]
+package-name = "kernel-6.1"
+patched-version = "kernel-6.1.94-99.176.amzn2023"
```

Comment:
I'm surprised there's just one instance of a CVE that applies to multiple kernels. I'd expect that to be more common if not the majority of cases.

Reply:
Agreed, so I verified myself manually - this is has become more common as the number of kernel CVEs began to increase; also, checking that past GHSAs on bottlerocket itself (specifically, those that came with the 1.20.3 release) also indicate this being the new norm. Only 3 of the 17 kernel advisories affected more than one kernel series: https://github.com/bottlerocket-os/bottlerocket/security/advisories